modules/examples/todolist/README.md

```diff
@@ -0,0 +1,24 @@
+# TodoList RPC example
+
+## Execution
+
+Running the server:
+
+```bash
+sbt example-todolist-server run
```

Comment:
This command didn't work for me, maybe you want:

`sbt "project example-todolist-server" "run"`

Reply:
My bad! Another option I was thinking of is `sbt example-todolist-server/run`